cookbooks/aws-parallelcluster-slurm/recipes/install/install_pyxis.rb

```diff
@@ -35,12 +38,59 @@
     set -e
     tar xf #{pyxis_tarball} -C /tmp
     cd /tmp/pyxis-#{pyxis_version}
-    CPPFLAGS='-I /opt/slurm/include/' make
-    CPPFLAGS='-I /opt/slurm/include/' make install
-    mkdir -p /opt/slurm/etc/plugstack.conf.d
-    echo -e 'include /opt/slurm/etc/plugstack.conf.d/*' | tee /opt/slurm/etc/plugstack.conf
-    ln -fs /usr/local/share/pyxis/pyxis.conf /opt/slurm/etc/plugstack.conf.d/pyxis.conf
+    CPPFLAGS='-I #{node['cluster']['slurm']['install_dir']}/include/' make
+    CPPFLAGS='-I #{node['cluster']['slurm']['install_dir']}/include/' make install
   PYXIS_INSTALL
   retries 3
   retry_delay 5
 end
+
+# Spank configurations
+
+directory node['cluster']['slurm']['spank_config_dir'] do
+  user 'root'
+  group 'root'
+  mode '0755'
+  recursive true
+end
+
+directory spank_examples_dir
+
+template "#{spank_examples_dir}/plugstack.conf" do
+  source 'pyxis/plugstack.conf.erb'
+  owner 'root'
+  group 'root'
+  mode '0644'
+end
+
+# Pyxis configurations
+
+directory node['cluster']['pyxis']['runtime_path'] do
```

Comment:
wondering if we want to create this directory on the AMI. what if the customer has their own pyxis configuration already?

Reply:
If user comes with his own Pyxis installation I think we should skip the whole recipe